keyboards/crkbd/rev1/split_scomm.c

```diff
@@ -63,9 +64,11 @@ int serial_update_buffers(int master_update)
         if( smatstatus == TRANSACTION_END ) {
             s_change_old = s_change_new;
 #ifdef CONSOLE_ENABLE
-            uprintf("slave matrix = %b %b %b %b\n",
+            if (debug_matrix) {
+                uprintf("slave matrix = %b %b %b %b\n",
                     serial_slave_buffer[0], serial_slave_buffer[1],
                     serial_slave_buffer[2], serial_slave_buffer[3]);
+            }
```

Comment:
dprintf should handle this automatically:

https://github.com/qmk/qmk_firmware/blob/d21730774752b8bc439564b1e3b687ad2aa0922c/tmk_core/common/debug.h#L67-L71


```suggestion
            dprintf("slave matrix = %b %b %b %b\n",
                serial_slave_buffer[0], serial_slave_buffer[1],	                    serial_slave_buffer[0], serial_slave_buffer[1],
                serial_slave_buffer[2], serial_slave_buffer[3]);	                    serial_slave_buffer[2], serial_slave_buffer[3]);
```